client-app-example/src/client.js

```diff
@@ -23,12 +26,19 @@ let iframeClient = new Client({
 // Add a listener that will handled config data passed from the host to the
 // client at startup.
 iframeClient.addListener('environmentalData', envData => {
-  const appLocale = envData.locale;
+  // Transform link URLs to match top-level app.
+  transformLinks();
 
+  const appLocale = envData.locale;
   const now = new Date();
   const localizedDate = new Intl.DateTimeFormat(appLocale).format(now);
   console.log(
-    `Got locale from host. Current date formatted for ${envData.locale} is: ${localizedDate}`
+    `Got locale from host. Current date formatted for ${
+      envData.locale
+    } is: ${localizedDate}`
+  );
```

Comment:
Looks like my editor's prettier plugin got ahold of this line and changed some formatting.